packages/core/src/local-admin-api.ts

```diff
@@ -0,0 +1,37 @@
+import { AdminApi, DiagnosticsLogger, LogStyle } from '@ceramicnetwork/common'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { LocalIndexApi } from './indexing/local-index-api.js'
+
+/**
+ * AdminApi for Ceramic core.
+ */
+export class LocalAdminApi implements AdminApi {
+  constructor(
+    private readonly indexApi: LocalIndexApi,
+    private readonly logger: DiagnosticsLogger
+  ) {}
+
+  async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
+    this.logger.log(LogStyle.info, `Local Admin Api will add model ids to index ${modelsIDs}`)
+    await this.indexApi.indexModels(modelsIDs)
+    this.logger.log(LogStyle.info, `Local Admin Api did add model ids to index ${modelsIDs}`)
+  }
+
+  getIndexedModels(): Promise<Array<StreamID>> {
+    return Promise.resolve(this.indexApi.indexedModels() ?? [])
+  }
+
+  async stopIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
+    this.logger.log(LogStyle.info, `Local Admin Api will remove model ids to index ${modelsIDs}`)
+    await this.indexApi.stopIndexingModels(modelsIDs)
+    this.logger.log(LogStyle.info, `Local Admin Api did remove model ids to index ${modelsIDs}`)
+  }
+
+  async replaceIndexedModels(modelsIDs: Array<StreamID>): Promise<void> {
+    this.logger.log(LogStyle.info, `Local Admin Api will replace model ids to index ${modelsIDs}`)
+    // TODO: Is this done in a single transaction? If not, it should be.
```

Comment:
I know we've discussed this a bunch, but I feel like we never had a proper resolution.  I still don't understand why we are building this "replace" endpoint at all if we have no plan to use it in our tools.  It's just increasing the complexity and surface area of our APIs, and giving us more things we need to test and maintain for little value as far as I can tell.  Can we please get rid of it?